open_parser/__init__.py

```diff
@@ -0,0 +1 @@
+from open_parser.base import OpenParser
```

Comment:
let's also add a __version__ = "0.0.1" as python __version__ standard for open source packages.